src/plugin.ts

```diff
@@ -17,17 +18,8 @@ export async function runPlugin(context: Context) {
     return await helloWorld(context);
   }
 
-  const contributors: Record<string, number> = {};
-
-  // increment the counter for each new event for every contributor in that issue or PR
-  const login = payload.sender?.login; // the user who triggered the event
-
-  if (login) {
-    if (contributors[login]) {
-      contributors[login]++;
-    } else {
-      contributors[login] = 1;
-    }
+  if (eventName === "issue_comment.created" && payload.comment.body.match(/rewards/i)) {
```

Comment:
limiting scope here isn't meaningful when it is originally desired to be called on all kind of events